packages/website-frontend/index.html

```diff
@@ -2,10 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Try out the Stryker dashboard, home for your test quality">
+    <link rel="icon" type="image/svg+xml" href="/src/assets/favicon.ico" />
```

Comment:
Shouldn't this link to a favicon in the `public` folder?

Reply:
It should, yes 👍